src/operator/tensor/square_sum-inl.h

```diff
@@ -53,13 +53,15 @@ inline bool SquareSumForwardInferStorageType(const nnvm::NodeAttrs& attrs,
   const auto& in_stype = in_attrs->at(0);
   auto& out_stype = out_attrs->at(0);
   bool dispatched = false;
-  if (!dispatched && in_stype == kRowSparseStorage && param.axis[0] == 1 && param.keepdims) {
+  const TShape axis = param.axis.has_value() ? param.axis.value() : TShape();
+  if (!dispatched && in_stype == kRowSparseStorage &&
+      axis.ndim() > 0 && axis[0] == 1 && param.keepdims) {
     // sum per row and keep dims
     dispatched = storage_type_assign(&out_stype, kRowSparseStorage,
                                      dispatch_mode, DispatchMode::kFComputeEx);
   }
-  if (!dispatched && in_stype == kRowSparseStorage &&
-      (param.axis[0] == 0 || (param.axis[0] == 1 && !param.keepdims))) {
+  if (!dispatched && in_stype == kRowSparseStorage && axis.ndim() > 0 &&
```

Comment:
I added extra check to make sure `axis != TShape()` when accessing axis[0] @reminisce